proxy/context.go

```diff
@@ -51,6 +51,7 @@ type context struct {
 	routeLookup          *routing.RouteLookup
 	cancelBackendContext stdlibcontext.CancelFunc
 	logger               filters.FilterContextLogger
+	skipperDuration      time.Duration
```

Comment:
It is hard to follow when this changes. I think we may consider adding a stop watch field, start it when we receive request, pause if around operations we do not want to measure and report elapsed time after we served the response, see this outline https://github.com/zalando/skipper/compare/metrics/stopwatch